{"name": "test_vwo_login_positive", "status": "broken", "statusDetails": {"message": "RecursionError: maximum recursion depth exceeded", "trace": "setup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"50dea8cb93dc01330a4de0064907e54c\")>\n\n    @allure.epic(\"VWO Login Test\")\n    @allure.feature(\"TC#1 - VWO App Positive Test\")\n    @pytest.mark.positive\n    def test_vwo_login_positive(setup):\n        driver = setup\n        loginPage = LoginPage(driver)\n        loginPage.login_to_vwo(usr=\"py2x@thetestingacademy.com\", pwd=\"Wingify@1234\")\n        time.sleep(10)\n        dashboardPage=DashboardPage(driver)\n        assert \"Dashboard\" in driver.title\n>       assert \"Aman\" in dashboardPage.get_user_logged_in_text()\n\ntestcases_VWO_login.py:44: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\pageObjects\\dashboardPage.py:18: in get_user_logged_in_text\n    return self.get_user_logged_in_text()\n..\\pageObjects\\dashboardPage.py:18: in get_user_logged_in_text\n    return self.get_user_logged_in_text()\nE   RecursionError: maximum recursion depth exceeded\n!!! Recursion detected (same locals & position)"}, "start": 1715827700355, "stop": 1715827711142, "uuid": "32485465-8357-4b93-a224-4a600cfd7549", "historyId": "66dc95d5e12947c6b49ffecd387acf0a", "testCaseId": "66dc95d5e12947c6b49ffecd387acf0a", "fullName": "tests.vwoLoginTests.testcases_VWO_login#test_vwo_login_positive", "labels": [{"name": "epic", "value": "VWO Login Test"}, {"name": "feature", "value": "TC#1 - VWO App Positive Test"}, {"name": "tag", "value": "positive"}, {"name": "parentSuite", "value": "tests.vwoLoginTests"}, {"name": "suite", "value": "testcases_VWO_login"}, {"name": "host", "value": "INDELNB1516"}, {"name": "thread", "value": "9576-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.vwoLoginTests.testcases_VWO_login"}]}